File: 002+003/002.py
#Nhap toa do 2 diem A, B
#tinh do dai vecto AB

from vector import do_dai_vecto

xA, yA = map(float, input("A(xA, yA)? ").split())
xB, yB = map(float, input("B(xB, yB)? ").split())

AB = do_dai_vecto(xA, yA, xB, yB)

print(f"|AB| = {AB}")

File: 002+003/002.rb
#Nhap toa do 2 diem A, B
#tinh do dai vecto AB

require_relative "vector"

print "A(xA, yA)? "
xa, ya = gets.split.map(&:to_f)

print "B(xB, yB)? "
xb, yb = gets.split.map(&:to_f)

ab = do_dai_vecto(xa, ya, xb, yb)

puts "|AB| = #{'%.4f' %ab}"
